build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -130,25 +145,33 @@
     # line below is sed'ed by build-server-docker script
     enablerepo: "pmm-server"
 
+- name: Create grafana config
+  include_role:
+    name: grafana
+
+- name: Install clickhouse
+  include_role:
+    name: clickhouse
+
+- name: Install nginx
+  include_role:
+    name: nginx
+
+- name: Install postgres
+  include_role:
+    name: postgres
```

Comment:
I think postgres should be installed before grafana

Reply:
I actually doesn't matter much because neither will run until everything is provisioned.

Reply:
ok